java/src/jmri/jmrit/beantable/SensorTableAction.java

```diff
@@ -501,8 +501,14 @@ void showPullUpChanged() {
         a.showPullUp(showPullUpBox.isSelected());
     }
 
+    void showStateChanged() {
```

Comment:
The current implementation of sensor tables *always* shows the sensor state. Why would I want to *not* show the current state? If this is not about showing the current state, change these field and method names to make it clearer what you are doing.

Reply:
I renamed this to showStateForgetAndQueryBoxChanged to be more accurate to what it does.